server/services/reporting.js

```diff
@@ -379,6 +385,38 @@ const getRosParticipantsReport = async () => {
     })
     .find(searchOptions);
 
+  if (region !== DEFAULT_REGION_NAME) {
+    const additionalEntries = await dbClient.db[collections.ROS_STATUS]
+      .join({
+        participantJoin: {
+          type: 'LEFT OUTER',
+          relation: collections.PARTICIPANTS,
+          on: {
+            id: 'participant_id',
+          },
+        },
+        siteJoin: {
+          type: 'LEFT OUTER',
+          decomposeTo: 'object',
+          relation: collections.EMPLOYER_SITES,
+          on: {
+            id: 'site_id',
+          },
+        },
+      })
+      .find({
+        status: 'assigned-new-site',
```

Comment:
I get now why there's 2 queries happening here (it's probably _possible_ to do it in a single query but no way is it less complicated) buuuut it took me a while to figure it out, I'm thinking a comment or two describing the business logic will save some headaches later if/when it has to be changed again.

Like, "we only want to report on participants who have been assigned-new-site if they have previously been assigned-same-site in the user's health region"

Reply:
Yeah, it's quite confusing. I'll add more comments!

Reply:
Could we save the additional query by conditionally setting `searchOptions` depending on the region? That seems to be the only difference between the two queries right?